.github/workflows/test.yaml

```diff
@@ -208,10 +209,81 @@ jobs:
 
   bench-script:
     runs-on: ubuntu-22.04
+    needs: test
     timeout-minutes: 60
     steps:
       - uses: actions/checkout@v4.1.3
-      - run: sudo ip a add 192.168.6.2/32 dev eth0
-      - run: hostname -I
-      - run: ./test/init_test.sh
-      - run: ~/bypass4netns/benchmark/run_bench.sh
+        # thanks to https://github.com/NobuoTsukamoto/my_actions_test/blob/main/.github/workflows/test_contains.yml
+      - name: cleanup
```

Comment:
Why do you need to clean up the disk?
For ENOSPC?

Reply:
Yes, due to ENOSPC. Without this, test images occupied GHA runner storage and  tests failed.